websites/R/Raven Code/presence.ts

```diff
@@ -14,74 +14,81 @@ presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: "logo",
 			details: "Geziniyor",
+			buttons: [{ label: "Sayfaya git", url: document.location.href }],
```

Comment:
This is bad practice

Reply:
why?

Reply:
You should have buttons specifically for each page where appropriate, not a "Go to page" button for every page

Reply:
ok, why?

Reply:
> ok, why?

Because that's low quality and too generic.

Reply:
That's how I want it to be.

Reply:
However, it seems that this is not how PreMiD wants it to be.

Reply:
Okay.